app/controllers/partials/PspSchemeDashboardPartialsController.scala

```diff
@@ -59,8 +57,7 @@ class PspSchemeDashboardPartialsController @Inject()(
               val futureSeqHtml = for {
                 schemeDetails <- schemeService.retrieveSchemeDetails(request.idOrException, idNumber, "srn")
                 schemeFs <- financialStatementConnector.getSchemeFS(schemeDetails.pstr)
-                interimDashboard <- featureToggleConnector.getNewPensionsSchemeFeatureToggle("interim-dashboard").map(_.isEnabled)
-                aftReturnsHtml <- pspDashboardAftReturnsPartial(interimDashboard, idNumber, schemeDetails.pstr, psaId)
+                aftReturnsHtml <- Future.successful(Html(""))
```

Comment:
Remove aftReturnsHtml entirely.
Should just yield scala.collection.immutable.Seq(paymentsAndChargesHtml)
I think this should work.

Reply:
removed

Reply:
removed